cmd/jujud/agent/machine/manifolds.go

```diff
@@ -167,6 +171,18 @@ type ManifoldsConfig struct {
 
 	// NewAgentStatusSetter provides upgradesteps.StatusSetter.
 	NewAgentStatusSetter func(apiConn api.Connection) (upgradesteps.StatusSetter, error)
+
+	// SingularFlagDuration defines for how long this agent will ask
+	// for controller administration rights.
+	SingularFlagDuration time.Duration
```

Comment:
The var name here is not very descriptive - when I saw it in the code above, I had to scroll to its definition to have any clue what it was for. Perhaps ControllerAdminAttemptDuration? Or something.

Reply:
renamed to ControllerLeaseDuration, hopefully that's more meaningful? ControllerAdminAttemptDuration says nothing to me